src/types/tx.ts

```diff
@@ -31,9 +31,9 @@ export type TxResult = TxError | TxSuccess;
 
 // //////////// ESTIMATE TX ////////////
 export type TxContent =
-  | { type: "cosmos"; val: EncodeObject[] }
+  | { type: "cosmos"; val: EncodeObject[]; attribute?: string }
   | { type: "terra"; val: Msg[]; wallet: ConnectedWallet }
-  | { type: "evm"; val: SimulContractTx | SimulSendNativeTx };
+  | { type: "evm"; val: SimulTx; log?: LogProcessor };
```

Comment:
Passing a "log processor" adds more flexibility than passing just an `attribute` as we do for Cosmos, as it allows us to process logs as we see fit per component. 

I know the current setup "works fine at the moment", but it forces us to:
- pass either `attribute` or `log` parameter to `TxContent` when we need some param read from the logs, the purpose of which is not immediately obvious unless you know how it will be used in the subsequent chain of functions (to return `TxSuccess.attrValue`)
- have different ways of populating `TxSuccess.attrValue` (for `cosmos` we pass `attribute` and it finds it for us, while for `evm` we pass the log filter function that the `sendTx` has to call)
- return `TxSuccess.attrValue`, the purpose of which is not immediately obvious (unless you know how it was populated)
- might not be a need for this atm, but this setup would stop us from being able to read multiple attributes from the logs

What do you think about just returning `TxSuccess.logs` instead of returning `TxSuccess.attrValue` and processing those logs as we see fit in specific functions/components? `.